index.js

```diff
@@ -1,52 +1,53 @@
-const Scope = require('./src/scope')
-const _ = require('./src/util/underscore.js')
-const assert = require('./src/util/assert.js')
-const tokenizer = require('./src/tokenizer.js')
-const statFileAsync = require('./src/util/fs.js').statFileAsync
-const readFileAsync = require('./src/util/fs.js').readFileAsync
+const Scope = require("./src/scope");
+const _ = require("./src/util/underscore.js");
+const assert = require("./src/util/assert.js");
+const tokenizer = require("./src/tokenizer.js");
+const statFileAsync = require("./src/util/fs.js").statFileAsync;
+const readFileAsync = require("./src/util/fs.js").readFileAsync;
 // const path = require('path')
-const url = require('./src/util/url.js')
-const Render = require('./src/render.js')
-const lexical = require('./src/lexical.js')
-const Tag = require('./src/tag.js')
-const Filter = require('./src/filter.js')
-const Parser = require('./src/parser')
-const Syntax = require('./src/syntax.js')
-const tags = require('./tags')
-const filters = require('./filters')
-const Promise = require('any-promise')
-const anySeries = require('./src/util/promise.js').anySeries
-const Errors = require('./src/util/error.js')
+const url = require("./src/util/url.js");
+const Render = require("./src/render.js");
+const lexical = require("./src/lexical.js");
+const Tag = require("./src/tag.js");
+const Filter = require("./src/filter.js");
+const Parser = require("./src/parser");
+const Syntax = require("./src/syntax.js");
+const tags = require("./tags");
+const filters = require("./filters");
+const Promise = require("any-promise");
+const anySeries = require("./src/util/promise.js").anySeries;
+const Errors = require("./src/util/error.js");
+const { checkVariableInComputation } = require("./validations");
```

Comment:
Didn't we decide to not add it here? 
We don't need to export it from here. We can export from validations.js and use it like

`import { blah } from "liquidjs/validations"`